.github/workflows/release.yml

```
@@ -60,13 +60,18 @@ jobs:
 
           if [ "${{ matrix.target }}" == "win-x64" ]; then
             7z a -tzip "${release_name}.zip" "./${release_name}/*"
+
+            # create .nupkg for NuGet
+            dotnet pack ThunderstoreCLI/ThunderstoreCLI.csproj -c Release -o "nuget/" -p:SelfContained=false -p:PublishTrimmed=false -p:PublishSingleFile=false -p:StartupObject="" -p:RuntimeIdentifier="" -p:PublishReadyToRun=false -p:PackAsTool=true
+            cp "nuget/*" "."
+            rm -r "nuget"
```

I don't know why I output it in a subfolder to begin with but if you are going to move the output anyway might as well just do it from the pack command (sorry for doing this twice, I couldn't figure out how multiline suggestions work 😂)
```suggestion
            dotnet pack ThunderstoreCLI/ThunderstoreCLI.csproj -c Release -o "." -p:SelfContained=false -p:PublishTrimmed=false -p:PublishSingleFile=false -p:StartupObject="" -p:RuntimeIdentifier="" -p:PublishReadyToRun=false -p:PackAsTool=true
```